kahuna/public/js/image/controller.js

```diff
@@ -170,6 +176,17 @@ image.controller('ImageCtrl', [
                 });
         };
 
+        ctrl.delete = function() {
+            const confirmed = window.confirm('Are you sure you want to delete this image from the Grid?');
+            if (confirmed) {
+                mediaApi.delete(image).then(() => {
+                    window.alert('The image will be deleted shortly!');
```

Comment:
You could probably extend [the above](https://github.com/guardian/grid/pull/1016/files#r36070673) to have a confirmation message.